2-mandatory/6-journey-planner.js

```diff
@@ -81,7 +84,12 @@ function getTransportModes() {}
 
   Hint: Use the corresponding array method to decide if an element is included in an array.
 */
-function isAccessibleByTransportMode() {}
+function isAccessibleByTransportMode(transportModes, mode) {
+  if(transportModes.includes(mode)){
```

Comment:
you can simplify it in one line, i.e.
```
return transportModes.includes(mode);
```
The ``includes()`` function returns either true or false